lib/diagnostic.rb

```diff
@@ -93,13 +98,13 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = 'arr.chomp(56,67)'
```

Comment:
So we can use `arr.pop 2` to get rid of the last two elements of an array even if we don't know how long the array is.